src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php

```diff
@@ -66,18 +68,27 @@ class CustomerAddressType extends AbstractType
      */
     private $contextCountryId;
 
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
     /**
      * @param TranslatorInterface $translator
      * @param ConfigurableFormChoiceProviderInterface $stateChoiceProvider
+     * @param $contextCountryId
+     * @param RouterInterface $router
      */
     public function __construct(
         TranslatorInterface $translator,
         ConfigurableFormChoiceProviderInterface $stateChoiceProvider,
-        $contextCountryId
+        $contextCountryId,
+        RouterInterface $router
```

Comment:
Just like in https://github.com/PrestaShop/PrestaShop/pull/19866 this is a BC break

So we have to check
- can we easily avoid the BC break
=> I dont think so 😅 
- then either we acknowledge the BC break or we find a workaround like creating a new class

Reply:
So following the same principle as in the other PR I am gonna leave the comment about BC break